addons/rose/addon/components/rose/form/fieldset/index.hbs

```diff
@@ -19,6 +19,16 @@
     </div>
   {{/if}}
 
+  {{#if @helperText}}
+      <Rose::Form::HelperText
+        @id='helper-text-{{this.id}}'
+        @error={{@error}}
+        @link={{@link}}
+        @linkText={{@linkText}}
+      >
+        {{@helperText}}
+      </Rose::Form::HelperText>
+    {{/if}}
```

Comment:
I don't think this will work exactly how we need it. Currently, `error`, `link`, and `linkText` are undefined and we just get a second line under the description. It seems like we should be updating the description block to include capabilities to handle use cases like a "learn more" link. Thoughts?

Reply:
Wouldn't those fields be passed in by the consumer of the component?
I think you wouldn't really use a description if you go with this route since the new `HelperText` includes text for that.

Perhaps I need more context but what capability does the description block even need? Couldn't you just have yielded the link yourself?

Is this to try and reuse the `HelperText` component?

Reply:
yes, the `helperText` will be used in place of the description as it includes the text and link. This practice is similar to how we handle `learn more` link in other fields. for ex: [input](https://boundary-ui-storybook.vercel.app/?path=/docs/rose-form-input--basic-field). 

<img width="1107" alt="Screen Shot 2022-09-19 at 12 33 17 PM" src="https://user-images.githubusercontent.com/15043878/191100706-f73b7281-83aa-4397-acb4-c81cc377cf74.png">

Reply:
Yes. I was thinking about this wrong. Thanks for walking me through it with more info!